src/Url/Providers/PublicFolder.php

```diff
@@ -21,7 +21,10 @@ class PublicFolder extends AbstractUrlProvider
      */
     public function getUrlFromPath($assetPath)
     {
+        // Ensure platform independency
+        $path = str_replace('\\', '/', $assetPath);
```

Comment:
The function is intended to generate a URL ... On windows DIRECTORY_SEPARATOR will yield a backslash that is not used in URLs.